<ui version="4.0" >
 <class>NetkitMainWindow</class>
 <widget class="QMainWindow" name="NetkitMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Visual Netkit</string>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>3</number>
    </property>
    <property name="leftMargin" >
     <number>3</number>
    </property>
    <property name="topMargin" >
     <number>3</number>
    </property>
    <property name="rightMargin" >
     <number>3</number>
    </property>
    <property name="bottomMargin" >
     <number>3</number>
    </property>
    <item>
     <widget class="QTabWidget" name="centralTabWidget" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGraphic" >
       <attribute name="title" >
        <string>Graphic laboratory</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="leftMargin" >
         <number>3</number>
        </property>
        <property name="topMargin" >
         <number>3</number>
        </property>
        <property name="rightMargin" >
         <number>3</number>
        </property>
        <property name="bottomMargin" >
         <number>3</number>
        </property>
        <item>
         <widget class="GraphicsView" name="graphicsView" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Labs" >
     <property name="title" >
      <string>Recent Labs</string>
     </property>
     <addaction name="separator" />
     <addaction name="actionClearMenu" />
    </widget>
    <addaction name="actionNewLab" />
    <addaction name="actionOpenLab" />
    <addaction name="menuRecent_Labs" />
    <addaction name="separator" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionCloseLab" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuGraph" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>Graph</string>
    </property>
    <widget class="QMenu" name="menuZoom" >
     <property name="title" >
      <string>Zoom</string>
     </property>
     <addaction name="actionZoomIn" />
     <addaction name="actionZoomOut" />
     <addaction name="actionZoomOriginal" />
    </widget>
    <widget class="QMenu" name="menuExportAs" >
     <property name="title" >
      <string>Export as...</string>
     </property>
     <addaction name="actionPNGImage" />
     <addaction name="actionSVGImage" />
    </widget>
    <widget class="QMenu" name="menuSize" >
     <property name="title" >
      <string>Scene size</string>
     </property>
     <addaction name="actionSmall" />
     <addaction name="actionNormal" />
     <addaction name="actionBig" />
     <addaction name="actionHuge" />
    </widget>
    <addaction name="menuZoom" />
    <addaction name="menuSize" />
    <addaction name="separator" />
    <addaction name="actionAddVirtualMachine" />
    <addaction name="actionAddCollisionDomain" />
    <addaction name="actionAddLink" />
    <addaction name="actionManageGraph" />
    <addaction name="separator" />
    <addaction name="menuExportAs" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Visual_Netkit" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuGraph" />
   <addaction name="menuView" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBarGeneralActions" >
   <property name="windowTitle" >
    <string>General Toolbar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewLab" />
   <addaction name="actionOpenLab" />
   <addaction name="actionSave" />
   <addaction name="actionSaveAs" />
   <addaction name="separator" />
   <addaction name="actionUndo" />
   <addaction name="actionRedo" />
  </widget>
  <widget class="QDockWidget" name="dockLab" >
   <property name="floating" >
    <bool>false</bool>
   </property>
   <property name="features" >
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle" >
    <string>Netkit Laboratory</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" >
     <property name="leftMargin" >
      <number>3</number>
     </property>
     <property name="topMargin" >
      <number>3</number>
     </property>
     <property name="rightMargin" >
      <number>3</number>
     </property>
     <property name="bottomMargin" >
      <number>3</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="labTree" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="columnCount" >
        <number>1</number>
       </property>
       <column>
        <property name="text" >
         <string>Lab</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockProperty" >
   <property name="allowedAreas" >
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle" >
    <string>Property Editor</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4" >
    <layout class="QVBoxLayout" >
     <property name="leftMargin" >
      <number>3</number>
     </property>
     <property name="topMargin" >
      <number>3</number>
     </property>
     <property name="rightMargin" >
      <number>3</number>
     </property>
     <property name="bottomMargin" >
      <number>3</number>
     </property>
     <item>
      <widget class="QTableWidget" name="propertyTable" >
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="showGrid" >
        <bool>true</bool>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <column>
        <property name="text" >
         <string>Property</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Value</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockMiniature" >
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle" >
    <string>Miniature</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6" >
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <property name="leftMargin" >
        <number>6</number>
       </property>
       <property name="topMargin" >
        <number>6</number>
       </property>
       <property name="rightMargin" >
        <number>6</number>
       </property>
       <property name="bottomMargin" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="labMiniature" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="minimumSize" >
          <size>
           <width>230</width>
           <height>230</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>230</width>
           <height>230</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="leftMargin" >
        <number>2</number>
       </property>
       <property name="topMargin" >
        <number>2</number>
       </property>
       <property name="rightMargin" >
        <number>2</number>
       </property>
       <property name="bottomMargin" >
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Zoom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="zoomSlider" >
         <property name="minimum" >
          <number>20</number>
         </property>
         <property name="maximum" >
          <number>200</number>
         </property>
         <property name="singleStep" >
          <number>10</number>
         </property>
         <property name="value" >
          <number>99</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition" >
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval" >
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="zoomLabel" >
         <property name="text" >
          <string>100%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockTreeView" >
   <property name="windowTitle" >
    <string>Tree View</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2" >
    <layout class="QVBoxLayout" >
     <property name="leftMargin" >
      <number>3</number>
     </property>
     <property name="topMargin" >
      <number>3</number>
     </property>
     <property name="rightMargin" >
      <number>3</number>
     </property>
     <property name="bottomMargin" >
      <number>3</number>
     </property>
     <item>
      <widget class="QTreeView" name="sceneTree" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockLog" >
   <property name="floating" >
    <bool>false</bool>
   </property>
   <property name="windowTitle" >
    <string>Logs</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3" >
    <layout class="QVBoxLayout" >
     <property name="leftMargin" >
      <number>3</number>
     </property>
     <property name="topMargin" >
      <number>3</number>
     </property>
     <property name="rightMargin" >
      <number>3</number>
     </property>
     <property name="bottomMargin" >
      <number>3</number>
     </property>
     <item>
      <widget class="QTextEdit" name="logText" >
       <property name="cursor" stdset="0" >
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking" >
        <bool>false</bool>
       </property>
       <property name="contextMenuPolicy" >
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="acceptDrops" >
        <bool>false</bool>
       </property>
       <property name="undoRedoEnabled" >
        <bool>false</bool>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarDrawing" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="windowTitle" >
    <string>Drawing Toolbar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomOut" />
   <addaction name="actionZoomOriginal" />
   <addaction name="actionZoomIn" />
   <addaction name="separator" />
   <addaction name="actionAddVirtualMachine" />
   <addaction name="actionAddCollisionDomain" />
   <addaction name="actionAddLink" />
   <addaction name="actionManageGraph" />
  </widget>
  <action name="actionNetkit_Lab" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Netkit Lab</string>
   </property>
  </action>
  <action name="actionMiniature" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Miniature</string>
   </property>
  </action>
  <action name="actionProperty_Editor" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Property Editor</string>
   </property>
  </action>
  <action name="actionTree_View" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Tree View</string>
   </property>
  </action>
  <action name="actionLog_Panel" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Log Panel</string>
   </property>
  </action>
  <action name="actionAbout_Visual_Netkit" >
   <property name="text" >
    <string>About Visual Netkit</string>
   </property>
  </action>
  <action name="actionNewLab" >
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/new_icon</iconset>
   </property>
   <property name="text" >
    <string>New Lab</string>
   </property>
  </action>
  <action name="actionOpenLab" >
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/open_icon</iconset>
   </property>
   <property name="text" >
    <string>Open Lab</string>
   </property>
  </action>
  <action name="actionClearMenu" >
   <property name="text" >
    <string>Clear Menu</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/save_icon</iconset>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/save_as_icon</iconset>
   </property>
   <property name="text" >
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionCloseLab" >
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/close_icon</iconset>
   </property>
   <property name="text" >
    <string>Close Lab</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/quit_icon</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
   <property name="menuRole" >
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/undo_icon</iconset>
   </property>
   <property name="text" >
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/redo_icon</iconset>
   </property>
   <property name="text" >
    <string>Redo</string>
   </property>
  </action>
  <action name="actionZoomIn" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/zoom_in_icon</iconset>
   </property>
   <property name="text" >
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomOut" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/zoom_out_icon</iconset>
   </property>
   <property name="text" >
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionZoomOriginal" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/zoom_original_icon</iconset>
   </property>
   <property name="text" >
    <string>Zoom original</string>
   </property>
  </action>
  <action name="actionAddVirtualMachine" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/vm_icon</iconset>
   </property>
   <property name="text" >
    <string>Add virtual machine</string>
   </property>
   <property name="autoRepeat" >
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAddCollisionDomain" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/cd_icon</iconset>
   </property>
   <property name="text" >
    <string>Add collision domain</string>
   </property>
  </action>
  <action name="actionPNGImage" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>PNG image</string>
   </property>
  </action>
  <action name="actionSVGImage" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>SVG image</string>
   </property>
  </action>
  <action name="actionSmall" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Small</string>
   </property>
   <property name="toolTip" >
    <string>Small (500x500)</string>
   </property>
  </action>
  <action name="actionNormal" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Normal</string>
   </property>
   <property name="toolTip" >
    <string>Normal (1000x1000)</string>
   </property>
  </action>
  <action name="actionBig" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Big</string>
   </property>
   <property name="toolTip" >
    <string>Big (2000x2000)</string>
   </property>
  </action>
  <action name="actionHuge" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Huge</string>
   </property>
   <property name="toolTip" >
    <string>Huge (4000x4000)</string>
   </property>
  </action>
  <action name="actionManageGraph" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/mouse_icon</iconset>
   </property>
   <property name="text" >
    <string>Manage graph</string>
   </property>
  </action>
  <action name="actionAddLink" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources/VisualNetkit.qrc" >:/menu/link_icon</iconset>
   </property>
   <property name="text" >
    <string>Add link</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>gui/GraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/VisualNetkit.qrc" />
 </resources>
 <connections/>
</ui>
